tests/test_patron.py

```diff
@@ -107,14 +180,34 @@ def test_populate_patron_common_fields_staff_no_krb_name_but_email_success(
     assert (
         patron_xml_record.user_identifiers.user_identifier.value.string == "222222222"
     )
+    assert (
+        patron_xml_record.find_all("user_identifier")[1].value.string
+        == "22222222222222"
+    )
 
 
 def test_populate_patron_common_fields_staff_null_values_success(
```

Comment:
Just noting that the null values tests will change when I add code to reject records, a record like this shouldn't pass successfully in  the completed app but this at least shows that the app won't error out when nothing but the `MIT_ID` is present